scripts/dev/templates/operator/Dockerfile.operator

```diff
@@ -0,0 +1,18 @@
+ARG imagebase
+FROM ${imagebase} AS builder
+
+FROM {{base_image}}
+
+ENV OPERATOR=manager \
+    USER_UID=1001 \
```

Comment:
Can we change this to `2000` and align with the SecurityContext we already have in the code? https://github.com/mongodb/mongodb-kubernetes-operator/blob/e8406136eea04ac52d48ac232f6ebd1104b76acc/pkg/kube/podtemplatespec/podspec_template.go#L167